pkg/cmd/cli/cmd/tag.go

```diff
@@ -138,6 +140,10 @@ func RunTag(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []stri
 			}
 		}
 	case "":
+		if len(ref.Registry) > 0 {
```

Comment:
Aliases are supposed to be image stream tags - alias to a docker image is
implicit

On Oct 21, 2015, at 3:09 AM, Michal Minar <notifications@github.com> wrote:

In pkg/cmd/cli/cmd/tag.go
<https://github.com/openshift/origin/pull/5176#discussion_r42589773>:

> @@ -138,6 +140,10 @@ func RunTag(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []stri
>  			}
>  		}
>  	case "":
> +		if len(ref.Registry) > 0 {

For an alias, I'd keep original behaviour ("DockerImage" being the
fallback).

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5176/files#r42589773>.